app/src/main/java/net/gsantner/markor/activity/DocumentActivity.java

```diff
@@ -66,13 +67,17 @@
 
     private static boolean nextLaunchTransparentBg = false;
 
-    public static void launch(Activity activity, File path, Boolean isFolder, Boolean doPreview, Intent intent) {
+
+    public static void launch(Activity activity, File path, Boolean isFolder, Boolean doPreview, Intent intent, final int... arg0lineNumber) {
```

Comment:
It seems the variadic `final int... arg0lineNumber` is not used for multiple values, but as a nullable integer. If this is the case, I suggest replacing it with `Integer lineNumber`, which can also be set to null

Reply:
changed